client/my-sites/plugins/plugin-details.jsx

```diff
@@ -308,7 +307,11 @@ function PluginDetails( props ) {
 			<QueryEligibility siteId={ selectedSite?.ID } />
 			<QuerySiteFeatures siteIds={ selectedOrAllSites.map( ( site ) => site.ID ) } />
 			<QueryProductsList persist={ ! wporgPluginNotFound } />
-			<FixedNavigationHeader compactBreadcrumb={ ! isWide } navigationItems={ breadcrumbs } />
+			<FixedNavigationHeader
+				compactBreadcrumb={ ! isWide }
+				navigationItems={ breadcrumbs }
+				className="plugins-browser__header"
```

Comment:
I needed to add the class name here so we can find the breadcrumbs on the Plugin Details page.

Reply:
Can we fix the test component instead? Sounds like it's leveraging the wrong class name or maybe `FixedNavigationHeader` needs a default class the test component can use.

Reply:
What is the concern? Is it the naming `plugins-browser__header` vs `plugin-details__header`? `FixedNaviationHeader` is used in other places, so we could put a default class name on it. Something like `fixed-navigation__header`? But I'm not sure I see the harm in providing a class name explicitly for the plugins pages.

I was thinking we could put a class name on the breadcrumb component itself, but it doesn't have a containing element and might be worthy of its own PR.

Reply:
CSS in calypso follows some sort of [BEM](https://css-tricks.com/bem-101/) (I don't know much more than that but the [docs](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md) might explain it better than I could).

`plugins-detail` is the component name that should be prefixing any classNames from this component. `plugins-browser` is a separate component and shouldn't be referenced in `plugins-details`.

My guess is that when creating tests that cover the `FixedNavigationHeader` usage we used a class specific to `plugins-browser` (probably what was under test). Instead we should have used something specific only to `FixedNavigationHeader` or shared between all the plugins pages at the very least. It may have made sense at the time but I think since we're expanding usage we should use something specific only to `FixedNavigationHeader`.

> What is the concern?

Main issue is that all tests covering `FixedNavigationHeader` seem to require adding an unrelated class just so the test code works. Test code for `FixedNavigationHeader` shouldn't dictate class name choices in the actual code, it should be reusable without this constraint.

Reply:
> Main issue is that all tests covering FixedNavigationHeader seem to require adding an unrelated class just so the test code works. Test code for FixedNavigationHeader shouldn't dictate class name choices in the actual code, it should be reusable without this constraint.

Ah, that makes sense. -- I added a default class [on the component itself](https://github.com/Automattic/wp-calypso/pull/67127/files#diff-d717431d31dbd940b8fb904c76f0d160e964e737d0f90780328884bcb38971d8R70): `fixed-navigation-header__header` and used that to find the breadcrumbs for testing. How does that look?

Reply:
LGTM!